packages/integrations/stores/ini-file/src/ini.ts

```diff
@@ -0,0 +1,44 @@
+import _ from 'lodash';
+import ini from 'ini';
+import { type Config } from '@configu/ts';
```

Comment:
Please replace the use of `@configu/ts` with `@configu/sdk`, make sure to remove the dependency from the `package.json` file.